lib/jazzy/assets/js/jazzy.js

```diff
@@ -1,3 +1,13 @@
+window.jazzy = {'docset': false}
```

Comment:
There was a reason why I put this in the HTML instead of the js file, just can't remember what it was